zip-draft-XXXX.rst

```diff
@@ -0,0 +1,289 @@
+2024 halving. The exact disbursement mechanism for the lockbox funds will be
+determined in a future ZIP, but the goal is to ensure:
+
+- Reduced regulatory risks by avoiding direct funding of specific
+  organizations.
+- Improved funding efficiency by allocating resources directly to projects
+  rather than organizations.
+- Enhanced decentralization by distributing decision-making power and ensuring
+  community involvement in funding decisions.
+- Potential tax efficiency by deferring taxable events until the liquidation of
+  funds by recipients, allowing for more flexibility in handling tax
+  liabilities and reducing the need for immediate liquidation compared to
+  directly funding US-based organizations with a USD bottom line.
+
+By addressing these issues, this proposal aims to ensure sustainable,
+efficient, and decentralized funding for essential activities within the Zcash
+ecosystem.
+
+Requirements
+============
+
+1. **In-Protocol Lockbox Creation**: The ZIP must define the creation of an
+   in-protocol lockbox where |percentage| of the Zcash block reward will be
+   allocated post-November 2024 halving. This lockbox will securely store funds
+   until a disbursement mechanism is established in a future ZIP.
+
+2. **Decentralization Goal**: The lockbox must be designed with the goal of
+   future decentralization in mind. While the disbursement mechanism will be
+   defined later, the lockbox should support the principles of decentralization
+   to ensure future community involvement and project-specific funding.
+
+3. **Regulatory Considerations**: The creation of the lockbox should minimize
+   regulatory risks by avoiding direct funding of specific organizations. The
+   design should ensure compliance with applicable laws and regulations to
+   support the long-term sustainability of the funding model.
+
+4. **Efficient Allocation Preparation**: The lockbox must be prepared to
+   allocate resources efficiently once the disbursement mechanism is defined.
+   This includes ensuring that funds are readily available for future projects
+   and not tied up in organizational overhead.
+
+5. **Potential Tax Efficiency**: The lockbox should be designed with the
+   potential to defer taxable events until the liquidation of funds by
+   recipients, allowing for more flexibility in handling tax liabilities
+   compared to direct funding of US-based organizations.
+
+6. **Security of Funds**: The lockbox must implement robust security measures
+   to protect against unauthorized access or misuse of funds. This includes
+   ensuring the integrity of the future decentralized voting/signature
+   mechanism.
+
+7. **Future Adaptability**: The lockbox design should be flexible and adaptable
+   to accommodate the disbursement mechanism defined in a future ZIP. This
+   ensures that the system can evolve based on feedback and new developments in
+   the Zcash ecosystem.
```

Comment:
I'm not sure these are _requirements_, since they couldn't possibly be fully addressed by the lockbox itself. I think it's fine to leave them in the motivation.